chart/crds/crd-awscodecommit.yaml

```diff
@@ -0,0 +1,182 @@
+# Copyright (c) 2020 TriggerMesh Inc.
```

Comment:
those CRD files cannot be links to the CRD in the /config directory of the repo ?

Reply:
good point. let me check

Reply:
symlink seem to work fine. a harmless message is printed while packaging the chart, but it looks good!